app/models/volunteer_op.rb

```diff
@@ -14,7 +14,6 @@ class VolunteerOp < ActiveRecord::Base
   scope :reachskills,          -> { where(source: 'reachskills') }
   scope :remote_only,          -> { where.not(source: 'local') }
 
```

Comment:
Should I change this file to its previous state without sorting methods by name? Or should I keep the sorting? Sorry for asking; before you commented I'd already returned to previous state without sorting and now just switched back. Anyway, I'll redo it as necessary.